deploy/README.md

```diff
@@ -66,29 +95,34 @@ CDK should be bootstrapped once, if this hasn't been done before.
 cdk bootstrap
 ```
 
-In order to deploy, check that the 
+In order to deploy, check that the
 stack synthesizes correctly and then deploy.
 
 ```sh
-cdk synth
-cdk deploy
+cdk synth --context "env=dev"
+cdk deploy --context "env=dev"
 ```
 
-Towards the end of the deployment you should get an API Gateway endpoint. This can be used with [awscurl] to query the htsget-rs server:
+### Testing the endpoint
+
+When the deployment is finished, the htsget endpoint can be tested by querying it. Since a JWT authorizer is used,
+a valid JWT token must be obtained in order to access the endpoint. This token should be obtained from AWS Cognito using
+the configured user pool id and audience parameters. Then `curl` can be used to query the endpoint:
 
 ```sh
-awscurl --region ap-southeast-2 https://<ID>.execute-api.ap-southeast-2.amazonaws.com/prod/reads/service-info
+curl -H "Authorization: <JWT Token>" "https://<htsget_domain>/reads/service-info"
 ```
 
 With a possible output:
+
 ```json
 {
   "id": "",
-  "name": "",
-  "version": "",
+  "name": "umccr-htsget-rs",
```

Comment:
Ditto here, perhaps UMCCR-specific details are better placed in infrastructure/wiki instead of this README?